sys/linux/socket_tipc_netlink.txt

```diff
@@ -170,6 +170,8 @@ tipc_nl_media_policy [
 tipc_nl_node_policy [
 	TIPC_NLA_NODE_ADDR	nlattr[TIPC_NLA_NODE_ADDR, int32]
 	TIPC_NLA_NODE_UP	nlattr[TIPC_NLA_NODE_UP, void]
+	TIPC_NLA_NODE_ID	nlattr[TIPC_NLA_NODE_ID, array[int8]]
+	TIPC_NLA_NODE_KEY	nlattr[TIPC_NLA_NODE_KEY, array[int8]]
```

Comment:
For TIPC_NLA_NODE_KEY, I think we can do better than `array[int8]`.  The kernel interprets this data as a `struct tipc_aead_key`.

Reply:
I created a new struct to hold the key. Thank you for the suggestion!